src/parser/smt2/Smt2.g

```diff
@@ -1703,7 +1697,36 @@ identifier[cvc5::ParseOp& p]
       }
     | sym=SIMPLE_SYMBOL nonemptyNumeralList[numerals]
       {
-        p.d_op = PARSER_STATE->mkIndexedOp(AntlrInput::tokenText($sym), numerals);
+        std::string opName = AntlrInput::tokenText($sym);
+        api::Kind k = PARSER_STATE->getIndexedOpKind(opName);
+        if (k==api::APPLY_SELECTOR || k==api::APPLY_UPDATER)
```

Comment:
```suggestion
        if (k == api::APPLY_SELECTOR || k == api::APPLY_UPDATER)
```

Reply:
Please add an example for `tuple_update` to one of the regressions like other tuple operators.